hooks/pre/31-upgrade-puppet.rb

```diff
@@ -1,20 +1,26 @@
-def stop_services                                                                                                                                                                                                                                     [0/1892]
+def puppet4_available?
+  success = []
+  success << Kafo::Helpers.execute('yum info puppet-agent')
+  success << Kafo::Helpers.execute('yum info puppetserver')
```

Comment:
ah cool, I was trying `yum search` instead of `yum info` and always getting zero exit code even if the pkg wasn't found. This change works as expected.